src/main/java/com/jcabi/log/Logger.java

```diff
@@ -355,6 +357,24 @@ public void write(final int data) throws IOException {
     }
 
     /**
+     * Set final static field.
+     * @param field Field
+     * @param value New value
+     * @throws NoSuchFieldException If some problem
+     * @throws IllegalAccessException If some problem
+     */
+    // @checkstyle JavadocLocationCheck (2 lines)
+    // @checkstyle ThrowsCountCheck (2 lines)
+    private static void setFinalStatic(final Field field, final Object value)
+        throws NoSuchFieldException, IllegalAccessException {
```

Comment:
@pecko let's catch `NoSuchFieldException, IllegalAccessException` and propagate them. something like this
```
catch (final NoSuchFieldException ex) {
    throw new IllegalStateException(ex)
}
```